tests/nlu/classifiers/test_sklearn_classifier.py

```diff
@@ -0,0 +1,72 @@
+import copy
+from typing import Text
+
+import pytest
+
+import rasa.shared.nlu.training_data.loading
+from rasa.engine.graph import ExecutionContext
+from rasa.engine.storage.resource import Resource
+from rasa.engine.storage.storage import ModelStorage
+from rasa.nlu import registry
+from rasa.nlu.classifiers.sklearn_intent_classifier import (
+    SklearnIntentClassifierGraphComponent,
+)
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.shared.nlu.training_data.training_data import TrainingData
+
+
+@pytest.fixture()
+def training_data(nlu_data_path: Text):
+    return rasa.shared.nlu.training_data.loading.load_data(nlu_data_path)
+
+
+@pytest.fixture()
+def default_sklearn_intent_classifier(
+    default_model_storage: ModelStorage, default_execution_context: ExecutionContext,
+):
+    return SklearnIntentClassifierGraphComponent.create(
+        SklearnIntentClassifierGraphComponent.get_default_config(),
+        default_model_storage,
+        Resource("sklearn"),
+        default_execution_context,
+    )
+
+
+def test_persist_and_load(
```

Comment:
can we add a test where loading the component from the model storage fails and we initialize an empty `SklearnClassifier`?